### ISSUE TITLE
Add MongoDB Service

### ISSUE THREAD

**Issue by kimar** [intercity/intercity#27]
Would be cool to have MongoDB support in Intercity.
Not sure if this is the right repo to file this issue though.

**Comment by jvanbaarsen** [comment]
Hi @kimar Thanks for the suggestion! I'll try to create some time next week to add MongoDB support!

**Comment by kimar** [comment]
Actually it would be really cool @jvanbaarsen to have some documentation around this / make it pluggable so that contributors can add those services more easily. I'd be happy to invest some of my spare time to actually work on this but haven't found any documentation around this.

**Comment by jvanbaarsen** [comment]
Yeah, documentation is something that needs a whole bunch of attention. I think for the time restraints I have right now it makes sense that I create the PR. But it would be great if you want to review it, and maybe we can then collaborate on docs on how to add a service.
